fuel-client/assets/schema.sdl

```diff
@@ -263,6 +263,26 @@ type FailureStatus {
 }
 
 
+type Genesis {
```

Comment:
Seems like we also need to add this to the cynic enums so that it doesn't use the fallback variant?

Reply:
Yep